config/mac_vars.yml

```diff
@@ -1,6 +1,7 @@
 oc_host_config_dir: /docker_shared/origin
 demo: True
+persistent_vol_mount_point: /persistedvolumes
```

Comment:
For Mac, instructions were to create a separate shared volume with a known name of /persistedvolumes

Still testing and want to determine exactly why we did this.  Putting this back to original value for now as I learn more.

Reply:
/persistedvolumes isn't going to work with 3.10 anymore. We don't have the option to set a hosted-pv-dirs flag. It will simply land under the base-dir.

If we want master to work with v3.9 and v3.10 we're going to have to add a whole bunch of logic to handle the different parameters and behavior, which was what we were trying to avoid by branching.

You may want to use the release-1.1 branch if you're trying to get v3.9 working.